app/src/main/java/at/aau/edu/wizards/MainViewModel.kt

```diff
@@ -15,6 +23,26 @@ class MainViewModel (private val sharedPreferences: SharedPreferences) : ViewMod
         sharedPreferences.edit().putString(SHARED_PREFERENCE_USERNAME_KEY, username).apply()
     }
 
+    fun getAvatar(): Int {
+        return sharedPreferences.getInt(SHARED_PREFERENCE_AVATAR_KEY, selectRandomAvatar())
```

Comment:
always random? Maybe it makes sense to keep preference of user

Reply:
This function is defined so, that the random avatar will be generated only the first time when the user starts the game. SharedPrefs are used here to remember the first choice the player made, which will be shown every other time he starts the game.